charms/docker/__init__.py

```diff
@@ -37,13 +36,45 @@ def running(self):
         # TODO: Add TCP:// support for running check
         return os.path.isfile(self.socket)
 
-    def run(self, image, options="", volumes=[], ports=[]):
-        volumes = " --volume=".join(volumes)
-        ports = " -p".join(ports)
-        cmd = "docker run {opts} {vols} {ports} {image}".format(
-            opts=options, vols=volumes, ports=ports, image=image)
+    def run(self, image, options=[], commands=[], arg=[]):
+        '''
+        Docker Run exposed as a method. This wont be as natural as the
+        command line docker experience.
+
+        Docker CLI output example:
+        Usage:	docker run [OPTIONS] IMAGE [COMMAND] [ARG...]
+
+        @param image - string of the container to pull from the registry,
+                        eg: ubuntu:latest
+        @param options - array of string  options, eg: ['-d', '-v /tmp:/tmp']
+        @param commands - array of string commands, eg: ['ls']
```

Comment:
Can you give the docker images more than one command?  I though it was just one.